tests/pages/firefox/os/tv.py

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class TVPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/os/devices/tv'
+
+    _next_button_locator = (By.CSS_SELECTOR, '.pager-next')
+    _previous_button_locator = (By.CSS_SELECTOR, '.pager-prev')
+    _screens_locator = (By.CSS_SELECTOR, '.pager-page')
+    _thumbnails_locator = (By.CSS_SELECTOR, '.pager-tabs a')
+
+    @property
+    def screens(self):
+        return [Screens(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._screens_locator)]
+
+    @property
+    def thumbnails(self):
+        return [Thumbnails(self.base_url, self.selenium, root=el) for el in
+                self.find_elements(self._thumbnails_locator)]
+
+    def click_next(self):
+        self.find_element(self._next_button_locator).click()
+
+    def click_previous(self):
+        self.find_element(self._previous_button_locator).click()
```

Comment:
I think these methods should be `show_next_screen` and `show_previous_screen` so we're describing the intent and not the specific interaction.